backtester/statistics/charts.py

```diff
@@ -0,0 +1,80 @@
+"""Generates charts from a portfolio report"""
+
+import altair as alt
+
+
+def returns_chart(report):
+    # Time interval selector
+    time_interval = alt.selection(type="interval", encodings=["x"])
+
+    # Area plot
```

Comment:
you could turn each comment + code block into a function, e.g. `def area_plot()`, `def nearest_point_selector()` etc